domain/learningandworkprogress/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewServiceTest.kt

```diff
@@ -118,90 +114,4 @@ class ReviewServiceTest {
       verify(reviewPersistenceAdapter).getCompletedReviews(PRISON_NUMBER)
     }
   }
-
-  @Nested
```

Comment:
We dont need these here as they are all covered in `ReviewServiceCreateReviewTest` and `ReviewServiceCalculateReviewWindowTest`